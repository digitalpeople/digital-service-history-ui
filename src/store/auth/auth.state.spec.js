import state from './auth.state';

describe('auth.state', () => {
  it('should match state', () => {
    expect(state).toMatchSnapshot();
  });
});
