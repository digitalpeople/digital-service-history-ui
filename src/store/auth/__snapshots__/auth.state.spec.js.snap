// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`auth.state should match state 1`] = `
Object {
  "isAuthenticated": false,
  "message": null,
  "userId": null,
}
`;
